cni/util/const.go

```diff
@@ -4,7 +4,8 @@ type ExecutionMode string
 
 // CNI execution modes
 const (
-	Default   ExecutionMode = "default"
-	Baremetal ExecutionMode = "baremetal"
-	AKSSwift  ExecutionMode = "aksswift"
+	Default       ExecutionMode = "default"
+	Baremetal     ExecutionMode = "baremetal"
+	AKSSwift      ExecutionMode = "aksswift"
+	V4OnlyOverlay ExecutionMode = "v4OnlyOverlay"
```

Comment:
instead of v4OnlyOverlay how about swiftv4Overlay or just v4Overlay?

Reply:
@neaggarwMS what do you think?

Reply:
Lets do v4Overlay

Reply:
Actually do we need v4Overlay? How about just Overlay? @tamilmani1989 ? DualStack option should come if we have v4 and v6 IPAM